<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QFrame" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>451</width>
     <height>556</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="createaccForm">
    <item row="0" column="0">
     <widget class="QLabel" name="acc_firstNameLabel">
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="acc_firstNameField">
      <property name="font">
       <font>
        <family>Proxima Nova</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(255, 255, 255);
background-color: rgb(120, 120, 120);
border:none;
border-radius: 5px;
padding-left: 10px;
margin-bottom:20px;
margin-top:10px;
}
QLineEdit::hover{
background-color: rgb(161, 161, 161);
}
QLineEdit::focus{
background-color: #474747
}</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="acc_lastNameLabel">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="acc_lastNameField">
      <property name="font">
       <font>
        <family>Proxima Nova</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(255, 255, 255);
background-color: rgb(120, 120, 120);
border:none;
border-radius: 5px;
padding-left: 10px;
margin-bottom:20px;
margin-top:10px;
}
QLineEdit::hover{
background-color: rgb(161, 161, 161);
}
QLineEdit::focus{
background-color: #474747
}</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="acc_contactNoLabel">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>Contact No.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="acc_contactNoField">
      <property name="font">
       <font>
        <family>Proxima Nova</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(255, 255, 255);
background-color: rgb(120, 120, 120);
border:none;
border-radius: 5px;
padding-left: 10px;
margin-bottom:20px;
margin-top:10px;
}
QLineEdit::hover{
background-color: rgb(161, 161, 161);
}
QLineEdit::focus{
background-color: #474747
}</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="acc_addressLabel">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="acc_addressField">
      <property name="font">
       <font>
        <family>Proxima Nova</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(255, 255, 255);
background-color: rgb(120, 120, 120);
border:none;
border-radius: 5px;
padding-left: 10px;
margin-bottom:20px;
margin-top:10px;
}
QLineEdit::hover{
background-color: rgb(161, 161, 161);
}
QLineEdit::focus{
background-color: #474747
}</string>
      </property>
      <property name="maxLength">
       <number>99</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="acc_emailLabel">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="acc_emailField">
      <property name="font">
       <font>
        <family>Proxima Nova</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(255, 255, 255);
background-color: rgb(120, 120, 120);
border:none;
border-radius: 5px;
padding-left: 10px;
margin-bottom:20px;
margin-top:10px;
}
QLineEdit::hover{
background-color: rgb(161, 161, 161);
}
QLineEdit::focus{
background-color: #474747
}</string>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="acc_usernameLabel">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="acc_usernameField">
      <property name="font">
       <font>
        <family>Proxima Nova</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(255, 255, 255);
background-color: rgb(120, 120, 120);
border:none;
border-radius: 5px;
padding-left: 10px;
margin-bottom:20px;
margin-top:10px;
}
QLineEdit::hover{
background-color: rgb(161, 161, 161);
}
QLineEdit::focus{
background-color: #474747
}</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="acc_passwordLabel">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="acc_passwordField">
      <property name="font">
       <font>
        <family>Proxima Nova</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(255, 255, 255);
background-color: rgb(120, 120, 120);
border:none;
border-radius: 5px;
padding-left: 10px;
margin-bottom:20px;
margin-top:10px;
}
QLineEdit::hover{
background-color: rgb(161, 161, 161);
}
QLineEdit::focus{
background-color: #474747
}</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="acc_confirmPasswordLabel">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>Confirm Password</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="acc_confirmPasswordField">
      <property name="font">
       <font>
        <family>Proxima Nova</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(255, 255, 255);
background-color: rgb(120, 120, 120);
border:none;
border-radius: 5px;
padding-left: 10px;
margin-bottom:20px;
margin-top:10px;
}
QLineEdit::hover{
background-color: rgb(161, 161, 161);
}
QLineEdit::focus{
background-color: #474747
}</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="acc_confirmPasswordLabel_6">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-bottom:20px;
margin-top:10px;

</string>
      </property>
      <property name="text">
       <string>Payment Method</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(157, 157, 157);
color: rgb(0, 0, 0);</string>
      </property>
      <item>
       <property name="text">
        <string>Cash on Delivery</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Master Card</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Visa Card</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Create Your Customer Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Register">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 226);
color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>680</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 226);
color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="Warninglbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>700</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 5, 38);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
